opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -231,6 +231,14 @@ namespace cpgrid
             // Make the grid.
             processEclipseFormat(g, nnc_cells, z_tolerance, false, turn_normals);
         }
+
+        {
+            const auto& ecl_actnum = ecl_grid.getACTNUM();
+            for (std::size_t g = 0; g < ecl_grid.getCartesianSize(); g++) {
```

Comment:
Using `g` for cell index is a little weird...? Other than that trivial observation, I'll try to run this through some cases to see if it is triggered.

Reply:
I like using `g` for `global_index` - happy to change it thougth.